client/src/main/scala/drt/client/components/TerminalDesksAndQueues.scala

```diff
@@ -99,8 +100,32 @@ object TerminalDesksAndQueues {
         else headings
       }
 
-      def subHeadingLevel2(queueNames: Seq[Queue]) = {
-        val queueSubHeadings = queueNames.flatMap(queueName => <.th(^.className := queueColour(queueName), "Pax") :: staffDeploymentSubheadings(queueName)).toTagMod
+
+      def staffDeploymentWithoutWaitColumnSubheadings(queueName: Queue): List[VdomTagOf[TableHeaderCell]] = {
```

Comment:
Most of this function is repeated code from the original function `staffDeploymentSubheadings`. We could refactor to move the common parts into their own function